lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1137,6 +1137,11 @@ bool PrimUpdaterManager::mergeToUsd(
     if (TF_VERIFY(hier)) {
         scene.notify(Ufe::SubtreeInvalidate(hier->defaultParent()));
     }
+
+    // We must recreate the UFE item because it has changed data models (Maya -> USD).
+    if (TF_VERIFY(ufeUsdItem))
+        scene.notify(Ufe::ObjectAdd(ufeUsdItem));
```

Comment:
I don't understand why this extra notif is required. The one above sends a subtree invalidate on the hier default parent which includes this item.

Reply:
I traced into why the sub-tree invalidation fails and it turns out it seems like a subtle bug in the code of the proxy-shape hierarchy implementation which result in UFE getting confused and failing to create a Hierarchy for a scene item. I'll try to see if it can be fixed instead.